apps/vmq_server/src/vmq_ranch.erl

```diff
@@ -361,25 +361,33 @@ internal_flush(#st{pending = Pending, socket = Socket} = State) ->
 %% Also note that the port has bounded buffers and port_command blocks
 %% when these are full. So the fact that we process the result
 %% asynchronously does not impact flow control.
-port_cmd(Socket, Data) ->
+tcp_send(Socket, Data) ->
     try
-        port_cmd_(Socket, Data),
+        tcp_send_(Socket, Data),
         ok
     catch
         error:Error ->
             {error, Error}
     end.
 
-port_cmd_({ssl, Socket}, Data) ->
+tcp_send_({ssl, Socket}, Data) ->
     case ssl:send(Socket, Data) of
         ok ->
             self() ! {inet_reply, Socket, ok},
             true;
         {error, Reason} ->
             erlang:error(Reason)
     end;
-port_cmd_(Socket, Data) ->
+tcp_send_(Socket, Data) ->
+    do_tcp_send(Socket, Data).
+
+-ifdef(tcp_send_erlang_port_command).
```

Comment:
You can use the macro `-if(?OTP_RELEASE >= 26).` here instead of defining a new compiler flag, it makes code easier to follow without having to look at the `rebar.config` file. All the VerneMQ supported OTP versions have that macro now

Reply:
What is the prefered way? I can change to the Release or keep the flag. Is a "speaking" flag prefered or the compiler version and a comment?

Reply:
@mths1 I'd prefer the release macro too. Looking forward to merging this.